script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -104,18 +104,18 @@ I've gotten so used to an ordinary house that living at a place like the Tohno m
 ".......To begin with,
 I can't imagine a Western-style building bigger than my school......"
 I'm amazed at the things I was able to endure as a child.
-The mere thought of being returned to such an anachronistic lifestyle makes me feel depressed, even if I don't feel anxious.
+The mere thought of returning to such an anachronistic lifestyle makes me feel depressed, even if I don't feel anxious.
 <Page29>
 The train arrives at a large station and stops for several minutes.
-Since this train stops at every station, I had to wait for the next rapid train to pass by.
+Since this train is stopping at every station, I have waited for the next rapid train to pass by.
 <Page30>
 There is no one else on the platform.
-At this time of day, I would expect to see a few businessmen in suits, but this morning it's particularly quiet.
+At this time of day, I was expecting to see a few businessmen in suits, but this morning has been particularly quiet.
 <Page31>
-An express train is passing by on the next track.
+An express train is passing on the next rail.
 "Thank you for waiting. The next stop is---"
 I hear a familiar announcement.
-The automatic door closes using pneumatic pressure.
+The automatic door closes up using pneumatic pressure.
```

Comment:
Another one that's a little hard to fully explain, but "closes up" seems more appropriate for something small that is sealed - like a clam, or small box. Dropping the `up` reads more fluently here (to me)